catalyst_voices_packages/catalyst_cardano_serialization/lib/src/address.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright 2021 Richard Easterling
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Why is copyright important?

Reply:
I've taken this class from https://github.com/reaster/cardano_wallet_sdk/blob/main/lib/src/address/shelley_address.dart

Reply:
Also cleaned up what we don't need and improved it too

Reply:
Anything thata Apache-2 or MIT is allowed.  Other licenses we need to review.
So, this license looks OK.